src/EditorFeatures/VisualBasicTest/TypeInferrer/TypeInferrerTests.vb

```diff
@@ -289,8 +289,14 @@ End Class</text>.Value
 
         <Fact, Trait(Traits.Feature, Traits.Features.TypeInferenceService)>
         <WorkItem(542837)>
-        Public Sub TestArrayRankSpecifier()
-            TestInMethod("Dim q As String() = New String([|Foo()|])", "System.Object")
+        Public Sub TestArrayRankSpecifier1()
+            TestInMethod("Dim q As String() = New String([|Foo()|])", "System.Char()")
```

Comment:
This is correct.  "String" doesn't have a constructor that takes an object.  It does have one that takes a char array.